README.md

```diff
@@ -437,7 +437,28 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 ## 8. Registering a new LLM provider
    Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
 
-## 9. Fine tuning
+## 9. TLS security profiles
+   TLS security profile can be set for the service itself and also for any configured provider. To specify TLS security profile for the service, the following section can be added into `ols` section in the `olsconfig.yaml` configuration file:
+
+```
+  tlsSecurityProfile:
+    type: OldType
```

Comment:
When setting the type to one of the 3 types OldType, IntermediateType, ModernType, is it still required to set the ciphers and minTLSVersion?

Reply:
I think it better to have defaults on known types and use Custom type to set ciphers and minTLSVersion

Reply:
no, it is optional

Reply:
yes @raptorsun